kevin_li/week01/day3/word_guesser/js/main.js

```diff
@@ -15,25 +15,34 @@ var guess = ['_', '_', '_'];  // line 5
 var word = ['F', 'O', 'X'];  // line 5
 
 var guessLetter = function (letter) {  // line 6, 7
+  var correct = false;
   for (var i = 0; i < guess.length; i++) {  // line 8
     if (letter == word[i]) {  // line 9
       guess[i] = letter; // line 9
       console.log(guess.join(' '));  // line 10
       console.log('Congratulation! You found a new letter!');  // line 10
-    } else if (guess[i] == '_') {
-      console.log('Sadness.');
-    } else if (guess[i] != '_') {
-      console.log('YOU HAVE WON!');
+      var correct = true;
     }
   }
-}
+  if (correct) {
+    if (guess[0] != '_' && guess[1] != '_' && guess[2] != '_') {  // line 11
```

Comment:
Interesting way to test for a win!

I still think a counter variable to count the correct guesses would be easier, i.e. `if(correctGuesses === word.length)`